<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3">
     <widget class="QPushButton" name="createButton">
      <property name="toolTip">
       <string>Create Test Cases</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/add-icon.png</normaloff>:/res/add-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="clearButton">
      <property name="toolTip">
       <string>Clear All Cases</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/Actions-edit-clear-icon.png</normaloff>:/res/Actions-edit-clear-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="connectButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Server Connection</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/connect-icon.png</normaloff>:/res/connect-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QPushButton" name="settingsButton">
      <property name="toolTip">
       <string>Settings</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/Categories-preferences-system-icon.png</normaloff>:/res/Categories-preferences-system-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>235</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="sendButton">
      <property name="toolTip">
       <string>Send Current Case</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/email-send-icon.png</normaloff>:/res/email-send-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="openButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>File Open</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/Open-file-icon.png</normaloff>:/res/Open-file-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="sendAllButton">
      <property name="toolTip">
       <string>Send All Cases</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/send_all_play_32.png</normaloff>:/res/send_all_play_32.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="13">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Param Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Send Set Message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Receive Set Message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Send Get Message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Receive Get Message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Set Error</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Get Error</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Set Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Get Description</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="pinButton">
      <property name="toolTip">
       <string>Send All Cases</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/show_detail.png</normaloff>:/res/show_detail.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="autoQueryButton">
      <property name="toolTip">
       <string>Send All Cases</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/query_start_32.png</normaloff>:/res/query_start_32.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
